_meta/ecs-migration.yml

```diff
@@ -0,0 +1,154 @@
+#   # Alias field is useful for fields where there is a 1-1 mapping from old to new
+#   alias: true-if-alias-is-required-in-6x (default is true)
+#   # Copy to is useful for fields where multiple fields map to the same ECS field
+#   copy_to: true-if-field-should-be-copied-to-target-in-6x (default is false)
+
+- from: beat.hostname
+  to: agent.server.host.name
+
+- from: beat.name
+  to: agent.server.type
+
+- from: beat.version
+  to: agent.server.version
+
+- from: meta.cloud.availability_zone
+  to: cloud.availability_zone
+
+- from: meta.cloud.instance_id
+  to: cloud.instance.id
+
+- from: meta.cloud.instance_name
+  to: cloud.instance.name
+
+- from: meta.cloud.machine_type
+  to: cloud.machine.type
+
+- from: meta.cloud.project_id
+  to: cloud.project.id
+
+- from: meta.cloud.provider
+  to: cloud.provider
+
+- from: meta.cloud.region
+  to: cloud.region
+
+- from: context.service.agent.name
+  to: agent.name
+
+- from: context.service.agent.version
+  to: agent.version
+
+- from: context.system.architecture
+  to: host.architecture
+
+- from: context.system.ip
+  to: host.ip
+
+- from: context.system.platform
+  to: host.os.platform
+
+- from: context.request.method
+  to: http.method
+
+- from: context.request.http_version
+  to: http.version
+
+- from: context.tags
+  to: labels
+  alias: false
+  copy_to: true
+
+- from: context.process.pid
+  to: process.pid
+
+- from: context.process.ppid
+  to: process.ppid
+
+- from: context.process.title
+  to: process.title
+
+  # not in ECS
+- from: context.service.environment
+  to: service.environment
+
+  # not in ECS
+- from: context.service.framework.name
+  to: service.framework.name
+
+  # not in ECS
+- from: context.service.framework.version
+  to: service.framework.version
+
+  # not in ECS
+- from: context.service.language.name
+  to: service.language.name
+
+  # not in ECS
+- from: context.service.language.version
+  to: service.language.version
+
+- from: context.service.name
+  to: service.name
+
+  # not in ECS
+- from: context.service.runtime.name
+  to: service.runtime.name
+
+  # not in ECS
+- from: context.service.runtime.version
+  to: service.runtime.version
+
+- from: context.request.url.full
+  to: url.full
+
+- from: context.request.url.hash
+  to: url.fragment
+
+- from: context.request.url.hostname
+  to: url.domain
+
+- from: context.request.url.pathname
+  to: url.path
+
+- from: context.request.url.port
+  to: url.port
+  alias: false
+  copy_to: true
+
+- from: context.request.url.raw
+  to: url.original
+
+- from: context.request.url.search
+  to: url.query
+
+- from: context.request.url.protocol
+  to: url.scheme
+  alias: false
+  copy_to: true
+
+- from: context.user.email
+  to: user.email
+
+- from: context.user.id
+  to: user.id
+
+- from: context.user.username
+  to: user.name
+
+- from: docker.container.id
+  to: container.id
+
+- from: docker.container.image
+  to: container.image.name
+
+- from: docker.container.name
+  to: container.name
```

Comment:
From non indexed fields we are also missing: 

`context.db.instance`, `context.db.statement`, `context.db.type`, `context.db.user`.
I suggest to map these to `database.instance`, `database.statement`, `database.type` and `database.user.name` as I cannot find any ECS definition, but we need to get rid of the `context`.

Reply:
Moved to https://github.com/graphaelli/apm-server/pull/1